src/System.Private.ServiceModel/tests/Unit/Description/ContractDescriptionTest.cs

```diff
@@ -1,143 +0,0 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
FYI -- @StephenBonikowsky and I chatted offline about these tests and decided to eliminate them because they use API's not in the contract.  It was an early attempt to unit test below the contract level, but of course it could not work in ToF or when we test against WCF packages instead of built binaries.  It was on my plate to delete this project anyway, so I'm happy to see this PR get rid of it.